tests/pytests/test_rdb_compatibility.py

```diff
@@ -73,6 +74,42 @@ def testRDBCompatibility(env):
         env.cmd('flushall')
         env.assertTrue(env.checkExitCode())
 
+def testRDBCompatibility_vecsim():
+    env = Env(moduleArgs='DEFAULT_DIALECT 2')
+    env.skipOnCluster()
+    skipOnExistingEnv(env)
+    dbFileName = env.cmd('config', 'get', 'dbfilename')[1]
+    dbDir = env.cmd('config', 'get', 'dir')[1]
+    rdbFilePath = os.path.join(dbDir, dbFileName)
+
+    rdbs = ['redisearch_2.4.14_with_vecsim.rdb']
```

Comment:
is it 2.4.14?

Reply:
I made an `.rdb` of version 2.4.14 for compatibility testing. It needs to be uploaded.